media/css/_social_buttons.scss

```diff
@@ -0,0 +1,42 @@
+.b-share {
+  a[data-service] {
+    -webkit-transition-property: color, text;
```

Comment:
Properties should be ordered color, -moz-transition-duration, -o-transition-duration, -webkit-transition-duration, -moz-transition-property, -o-transition-property, -webkit-transition-property, -moz-transition-timing-function, -o-transition-timing-function, -webkit-transition-timing-function